### PR TITLE
Task/setup bulk es load

### PR DESCRIPTION
**Pull request by BlanchardMR**
added the basic structure for bulk load into elastic search. made some other minor adjustments. added in dependency for django-elasticsearch-dsl-drf

### CODE REVIEW

**Comment by bragu** [comment]
I think Mona added a revised README.md that includes view creation and how to pass parameters to API. So, api/README.md may not be required.

**Review by elarens — APPROVED** [review]
I don't know enough about this part of the project to make informed comments, so not blocks or concerns from me.

**Comment by nataliemona** [comment]
> I think Mona added a revised README.md that includes view creation and how to pass parameters to API. So, api/README.md may not be required.

@BlanchardMR Your branch is just not up to date. You should update it.

File: backend/core/settings.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 # Elastic Search
 # https://django-elasticsearch-dsl.readthedocs.io/en/latest/quickstart.html
 ELASTICSEARCH_DSL = {
-    "default": {"hosts": "localhost:9200"},
+    "default": {"hosts": "host.docker.internal:9200"},
```
Comment (nataliemona) [inline backend/core/settings.py:39]: Why did you need to change this? Did localhost not work?

Comment (bragu) [reply backend/core/settings.py:39]: Yes... localhost refers to only local django container. Since we are referring to es container, we need to indicate that as "host.docker.internal"

Comment (nataliemona) [reply backend/core/settings.py:39]: Oh, I think I see what's wrong. The problem is that I've set up react, django and pgdb to be on the default bridge network and I've set elasticsearch and kibana to be on a separate bridge network. So django and elasticsearch can't communicate inside docker because they are on different networks.

`host.docker.internal` is for the container contacting the host... in this case I think you had to change it to this because django can't reach elasticsearch through the network inside docker so it has to access elasticsearch via the host

I need to fix this and put them all on the same network. But I guess you can do this for now and I when I fix it I will change this. I think the host should actually be  `elasticsearch:9200`

Comment (nataliemona) [reply backend/core/settings.py:39]: I'm glad I asked because I wouldn't have realized this otherwise. Fix is in my PR.

Comment (BlanchardMR) [reply backend/core/settings.py:39]: ok sounds good

File: backend/ctgov/api/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+#### Prerequisite
+
```
Comment (nataliemona) [inline backend/ctgov/api/README.md]: Can you remove this file?

File: backend/core/settings.py [hunk 3]
```diff
@@ -50,7 +50,8 @@
     "django.contrib.staticfiles",
     "rest_framework",
     "ctgov",
-    "corsheaders",
```
Comment (nataliemona) [inline backend/core/settings.py:53]: Add this back

File: docker-compose.yml [hunk 4]
```diff
@@ -61,6 +61,8 @@ services:
       - elsdata:/usr/share/elasticsearch/data
     networks:
       - elastic
+    logging:
+      driver: "none"
```
Comment (nataliemona) [inline docker-compose.yml]: Can you take this out? (there's better ways to not see logs)

File: requirements-dev.txt [hunk 5]
```diff
@@ -1,4 +1,5 @@
 pre-commit
 black
 flake8
-django>=3.1.7
\ No newline at end of file
+django>=3.1.7
```
Comment (nataliemona) [inline requirements-dev.txt]: Remove the django dependencies from this file... they aren't dev requirements. Dev requirements are related to testing or linting (or anything not related to building and running the project)

File: backend/core/settings.py (hunk 3 above)
Comment (BlanchardMR) [reply backend/core/settings.py:53]: fixed

**Review by nataliemona — APPROVED** [review]
